android/app/src/main/java/updated/mysterium/vpn/common/data/WalletEstimatesUtil.kt

```diff
@@ -0,0 +1,63 @@
+package updated.mysterium.vpn.common.data
+
+import mysterium.Estimates
+import network.mysterium.ui.UnitFormatter
+
+object WalletEstimatesUtil {
+
+    private const val MIN = "min"
+    private const val HOUR = "h"
+
+    fun convertVideoData(estimates: Estimates) = if (estimates.videoMinutes < 60) {
+        estimates.videoMinutes.toString()
+    } else {
+        (estimates.videoMinutes / 60).toString()
+    }
+
+    fun convertVideoType(estimates: Estimates) = if (estimates.videoMinutes < 60) {
+        MIN
+    } else {
+        HOUR
+    }
+
+    fun convertWebData(estimates: Estimates) = if (estimates.browsingMinutes < 60) {
+        estimates.browsingMinutes.toString()
+    } else {
+        (estimates.browsingMinutes / 60).toString()
+    }
+
+    fun convertWebType(estimates: Estimates) = if (estimates.browsingMinutes < 60) {
+        MIN
+    } else {
+        HOUR
+    }
+
+    fun convertDownloadData(estimates: Estimates): String {
+        val downloadData = UnitFormatter.bytesDisplay(estimates.trafficMB * 1024 * 1024)
+        return downloadData.value
+            .replace(',', '.')
+            .toDouble()
+            .toLong()
+            .toString()
+    }
```

Comment:
make this function in more efficient way

Reply:
Fixed